shared/src/main/scala/amf/core/internal/metamodel/document/BaseUnitProcessingDataModel.scala

```diff
@@ -15,14 +15,21 @@ trait BaseUnitProcessingDataModel extends ModelDefaultBuilder {
                "transformed",
                "Indicates whether a BaseUnit was transformed with some pipeline")
   )
+
+  val SourceSpec: Field =
+    Field(
+        Str,
+        ApiContract + "sourceSpec",
```

Comment:
field iri and model doc do not match, don't see much of a problem right now.

Reply:
fixed